### PR TITLE
[PowerPoint] (apis) GA of insert slides and req set 1.2

### PR DESCRIPTION
**Pull request by Rick-Kirkham**
There will be a parallel PR, with the same branch name, in the -reference repo. They should publish roughly in sync so that the ToCs are integrated.

### CODE REVIEW

File: docs/reference/requirement-sets/powerpoint-api-1-2-requirement-set.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+| Feature area | Description | Relevant objects |
+|:--- |:--- |:--- |
+| Insert and Delete Slides | Allows the insertion of existing slides into the current presentation from another presentation, as well as the ability to delete slides. | [Slide.delete](/javascript/api/powerpoint/powerpoint.slide#delete--), [Presentation.insertSlidesFromBase64](/javascript/api/powerpoint/powerpoint.presentation#insertslidesfrombase64-base64file--options-)|
+
+## API list
+
+The following table lists the PowerPoint JavaScript API requirement set 1.2. For a complete list of all PowerPoint JavaScript APIs (including preview APIs and previously released APIs), see [all PowerPoint JavaScript APIs](/javascript/api/powerpoint?view=powerpoint-js-preview&preserve-view=true).
+
+| Class | Fields | Description |
```
Comment (AlexJerabek) [inline docs/reference/requirement-sets/powerpoint-api-1-2-requirement-set.md:23]: Was this hand-authored or pulled from https://github.com/OfficeDev/office-js-docs-reference/tree/master/docs/requirement-set-tables?

File: docs/reference/requirement-sets/powerpoint-api-1-1-requirement-set.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
+title: PowerPoint JavaScript API requirement set 1.1
+description: 'Details about the PowerPointApi 1.1 requirement set.'
+ms.date: 12/04/2020
```
Comment (AlexJerabek) [inline docs/reference/requirement-sets/powerpoint-api-1-1-requirement-set.md:4]: Update the date!
```suggestion
ms.date: 01/08/2021
```

File: docs/reference/requirement-sets/powerpoint-api-1-2-requirement-set.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
+---
+title: PowerPoint JavaScript API requirement set 1.2
+description: 'Details about the PowerPointApi 1.2 requirement set.'
+ms.date: 12/04/2020
```
Comment (AlexJerabek) [inline docs/reference/requirement-sets/powerpoint-api-1-2-requirement-set.md:4]: Update the date!
```suggestion
ms.date: 01/08/2021
```

File: docs/reference/requirement-sets/powerpoint-api-1-2-requirement-set.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+||[id](/javascript/api/powerpoint/powerpoint.slide#id)|Gets the unique ID of the slide.|
+|[SlideCollection](/javascript/api/powerpoint/powerpoint.slidecollection)|[getCount()](/javascript/api/powerpoint/powerpoint.slidecollection#getcount--)|Gets the number of slides in the collection.|
+||[getItem(key: string)](/javascript/api/powerpoint/powerpoint.slidecollection#getitem-key-)|Gets a slide using its unique ID.|
+||[getItemAt(index: number)](/javascript/api/powerpoint/powerpoint.slidecollection#getitemat-index-)|Gets a slide using its zero-based index in the collection.|
+||[getItemOrNullObject(id: string)](/javascript/api/powerpoint/powerpoint.slidecollection#getitemornullobject-id-)|Gets a slide using its unique ID.|
+||[items](/javascript/api/powerpoint/powerpoint.slidecollection#items)|Gets the loaded child items in this collection.|
+
+## See also
+
+- [PowerPoint JavaScript API Reference Documentation](/javascript/api/powerpoint?view=powerpoint-js-preview&preserve-view=true)
```
Comment (AlexJerabek) [inline docs/reference/requirement-sets/powerpoint-api-1-2-requirement-set.md:40]: This should point to the view for PPT 1.2.
```suggestion
- [PowerPoint JavaScript API Reference Documentation](/javascript/api/powerpoint?view=powerpoint-js-1.2&preserve-view=true)
```

File: docs/reference/requirement-sets/powerpoint-api-requirement-sets.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
 ---
 title: PowerPoint JavaScript API requirement sets
 description: 'Learn more about the PowerPoint JavaScript API requirement sets.'
-ms.date: 10/26/2020
+ms.date: 12/04/2020
```
Comment (AlexJerabek) [inline docs/reference/requirement-sets/powerpoint-api-requirement-sets.md:4]: Update the date!
```suggestion
ms.date: 01/08/2021
```

File: docs/toc.yml [hunk 6]
```diff
@@ -1090,9 +1090,15 @@ items:
...
         href: reference/requirement-sets/powerpoint-api-requirement-sets.md
         displayName: PowerPoint
-      - name: PowerPoint preview APIs
+      - name: PowerPointApi Preview requirement set
         href: reference/requirement-sets/powerpoint-preview-apis.md
         displayName: PowerPoint
+      - name: PowerPointApi 1.1 requirement set
+        href: reference/requirement-sets/powerpoint-api-1-1-requirement-set.md
+        displayName: PowerPoint
+      - name: PowerPointApi 1.2 requirement set
```
Comment (AlexJerabek) [inline docs/toc.yml:1099]: Excel and Word have the order of their requirement set pages flipped (e.g.: 1.2 is above 1.1). We should reconcile that difference one way or the other, but I'd recommend sticking with that convention.

File: docs/reference/requirement-sets/powerpoint-api-1-2-requirement-set.md (hunk 1 above)
Comment (Rick-Kirkham) [reply docs/reference/requirement-sets/powerpoint-api-1-2-requirement-set.md:23]: @AlexJerabek I copied it from the old preview req set article which you made in October when I was out. It looks like you copied it from that file you linked to.

**Review by AlexJerabek — APPROVED** [review]

**Pull request merged**